specification/configuration.md

```diff
@@ -130,6 +136,32 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+### Real User Monitoring Libraries
+
+Real User Monitoring (RUM) instrumentation libraries cannot use environment variables for configuration.
+Instead, they MUST expose a `SplunkRum` object/class/namespace (depending on the language used) that allows
+setting the following properties (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.):
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 | Required |
+| -------------------------------------- | -----------                                                                                                                                                                                                 | -------- |
+| `realm` ()                             | Splunk realm, e.g. `us0`, `us1`. If set, value of `beaconUrl` will be automatically computed based on this. [1] [2]                                                                                         | Yes      |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. If both `realm` and `beaconUrl` are set, `beaconUrl` takes precedence. [1] [2]                                                        | Yes      |
+| `rumAuth` ()                           | RUM authentication token. [1]                                                                                                                                                                               | Yes      |
```

Comment:
Same question here -- we use the term "access token" for ingest and in UI you can only define these under "access token". Totally agreed that we need to distinguish the RUM token, but perhaps we can make the name consistent.

```suggestion
| `rumAccessToken` ()                           | RUM access token. [1]                                                                                                                                                                               | Yes      |
```

Reply:
`rumAccessToken` is what I had been using in the android configuration, so I in favor of that. :)

Reply:
hrm. no, I was using `rumAuthToken`. Either way, Android would have to change (really not a big deal, though, whatever we decide on).

Reply:
+1 for this as well :)

Reply:
There was some discussion on this: https://github.com/signalfx/gdi-specification/pull/112#discussion_r657313646

I agree that "access token" is more consistent with UI, but distinguishing RUM tokens from other tokens also makes sense.